lms/templates/ccx/schedule.html

```diff
@@ -32,7 +32,7 @@
 <section id="enter-date-modal" class="modal" aria-hidden="true">
   <div class="inner-wrapper" role="dialog">
     <button class="close-modal">
-      <i class="fa-remove"></i>
+      <i class="fa fa-times"></i>
```

Comment:
fa-time is font awesome class for cross icon, edx is using font awesome , I was not able to find  fa-remove in css files

http://fortawesome.github.io/Font-Awesome/icon/times/